Resources/engineCommandPerms.yml

```diff
@@ -65,15 +123,20 @@
 - Flags: SPAWN
   Commands:
   - spawn
+  - cspawn
 
 - Flags: HOST
   Commands:
   - gc_mode
   - gc
+  - gcf
   - loglevel
   - saveconfig
   - testlog
   - sudo
+  - scsi
+  - csi
```

Comment:
I don't think this should be restricted, since it already requires a build with `CLIENT_SCRIPTING`

Reply:
Because of the above, I think this should be `Debug` at max, or frankly, unrestricted. If anyone manages to get a build with it `CLIENT_SCRIPTING` on a public server, they've almost certainly undone any permissions checks.